networks/suzuka/suzuka-full-node/src/manager.rs

```diff
@@ -0,0 +1,43 @@
+use crate::SuzukaFullNode;
+use anyhow::Context;
+use super::partial::SuzukaPartialNode;
+use godfig::{
+    Godfig,
+    backend::config_file::ConfigFile
+};
+use suzuka_config::Config;
+use maptos_dof_execution::v1::Executor;
+
+#[derive(Clone)]
+pub struct Manager<Dof>
+    where 
+    Dof : SuzukaFullNode {
+    godfig: Godfig<Config, ConfigFile>,
+    _marker : std::marker::PhantomData<Dof>,
+}
+
+// Implements a very simple manager using a marker strategy pattern.
+impl Manager<SuzukaPartialNode<Executor>> {
+    pub async fn new(file : tokio::fs::File) -> Result<Self, anyhow::Error> {
+        let godfig = Godfig::new(ConfigFile::new(file), vec![]);
+        Ok(Self {
+            godfig,
+            _marker: std::marker::PhantomData,
+        })
+    }
+
+    pub async fn try_run(&self) -> Result<(), anyhow::Error> {
```

Comment:
I think it's fine to name it `run`, there is no name convention that says you should distinguish all fallible methods with a `try_` prefix.

Also perhaps, pass the recipient as `self` to not constrain the implementation into interior mutability.